### ISSUE TITLE
Another accept task is ongoing

### ISSUE THREAD

**Issue by Jax-p** [denoland/deno#6301]
I've just started with `Deno v1.1`.  
I've tried to create my first class and whenever I try to instance it I get:   
 `Uncaught Error: Another accept task is ongoing at unwrapResponse ($deno$/ops/dispatch_json.ts:43:11)`
It happens only if I use MySQL client before but It works if I run any MySQL query before `new MyClass`.

MyClass.ts
```javascript
class MyClass {
   // empty
}
export default MyClass;
```

server.ts
```javascript
import { Application } from "https://deno.land/x/oak@v5.2.0/mod.ts"
import { Client } from "https://deno.land/x/mysql/mod.ts";
import { APP_HOST, APP_PORT } from "./config.ts";
import router from './routes.ts'
import MyClass from "./controllers/myclass.ts";

const app = new Application()

app.use(router.routes())
app.use(router.allowedMethods())

const db = await new Client().connect({
    //...
});

const myClass = new MyClass();
await app.listen(`${APP_HOST}:${APP_PORT}`);
```
I use theese flags:
```
deno run --allow-env --allow-net server.ts
```

----
### Env
Linux Mint 19,
Deno 1.1

**Comment by ry** [comment]
Thanks for the report. It would be very useful if you can narrow this down to a small example that uses only the Deno API.

**Comment by Jax-p** [comment]
I've been trying to clarify this situation but I am not sure about it at all. As I've tested it can be corrected by banal adjustments that are not related to each other. It works if do at least one of these:
- remove `${APP_HOST}:${APP_PORT}` from last `console.log()`
- call any MySQL query before `new MyClass()`
- don't instance any class
- remove any method (which is not even used) from _products.ts_
 
I can't provide minimal reproducible code because I am really not sure whats going on.
So if you are interested I've created repository with whole code used and sent invitation to you.

In my another Deno projects everything works fine so I am not really in need of getting this working but it might be useful information for you. Btw. It was my first test and it doesn't work at all. I've just provided everything needed for the error.

Whole log:
```
error: Uncaught Error: Another accept task is ongoing
    at unwrapResponse ($deno$/ops/dispatch_json.ts:43:11)
    at Object.sendAsync ($deno$/ops/dispatch_json.ts:98:10)
    at async ListenerImpl.accept ($deno$/net.ts:63:17)
    at async Server.acceptConnAndIterateHttpRequests (https://deno.land/std@0.57.0/http/server.ts:216:14)
    at async MuxAsyncIterator.callIteratorNext (https://deno.land/std@0.57.0/async/mux_async_iterator.ts:28:29)

```
